### DISCUSSION TITLE
Welcome to capacitor-app-update Discussions!

### DISCUSSION THREAD

**Discussion by robingenz** [capawesome-team/capacitor-app-update#66] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by hishamHafez** [comment]


**Reply by robingenz** [reply]
Yes, take a look at [Test with internal app-sharing](https://github.com/capawesome-team/capacitor-app-update#test-with-internal-app-sharing).
